prov/tcp/src/tcpx_progress.c

```diff
@@ -374,157 +318,68 @@ static void process_pe_lists(struct tcpx_ep *ep)
 	process_tx_pe_entry(pe_entry);
 }
 
+/* todo this function will be cleaned up with send/recv changes */
+static void process_rx_requests(struct tcpx_ep *ep)
```

Comment:
This function is more for the short term until I complete the optimization for send and recv functions. Currently the send/recv calls don't propagate the errors to CQ which also will be fixed in next commit